en/docs/tutorials/scenarios/scenario10-notifications-webhooks.md

```diff
@@ -1,4 +1,62 @@
 # Scenario 10 - Notifications Using Webhooks
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**From time to time GoGo transit provides notifications to the Train companies. (For eg: platform unavailability, revised fees etc). GoGo transit is planning to provide these information as webhooks so that the Train companies can subscribe to these notifications without continuously polling.**
+
+WebHooks only allows one-way communication, from a caller web app to calle web app. The client who intends 
+to receive the events from the server/web app/publisher has to register its URL against the interesting events 
+in the publisher. When an event occurs, if a client is registered for that event the publisher will make an 
+HTTP POST request to the registered URL of the client to notify the event.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/scenario_notification_api.png" alt="Webhook notifications" title="Webhook notifications" width="60%" />
+
+You can view Webhook API created in WSO2 API manager by going to the publisher 
+(Super tenant - admin). [https://localhost:9443/publisher/apis](https://localhost:9443/publisher/apis) 
+
+You can publish notifications under different topics.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/notification_api_topics.png" alt="Notification Topics" title="Notification Topics" width="60%" />
+
+To subscribe to the API, you need to go to the developer portal.
+
+1. Go to [https://localhost:9443/devportal/](https://localhost:9443/devportal/) Dev portal and select **Carbon.super** tenant domain. This will redirect you to the super tenant developer portal.
+2. Sign in with a Super tenant, developer portal user. (peter)
+3. Click on NotificationAPI and click subscribe using a policy and generate the access token.
+4. To receive notifications, we need to register a service which will be called for each event. We can use [https://webhook.site/](https://webhook.site/) for this purpose. You can use the webhook site unique url to subscribe yourself to the **general** topic. 
+5. Go to **Try out** tab and enter the following details under **general** topic.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/notification_subscribe_devportal.png" alt="Subscribing to Topic" title="Subscribing to Topic" width="60%" />
+
+Click on **Generate Curl** and copy the Curl the command and run it on your terminal. 
+This will subscribe your client (webhook site) to the API manager. You can verify that the subscription was 
+successful by checking the event that was received on the Webhook site.
+
+
+6. To send a notification, you need to retrieve the callback URL for a topic from the publisher. Sign in to the publisher again with admin user credentials and select NotificationsAPI.
+7. You can get the Callback URL, by visiting the Topics tab and selecting the topic you desire. Since we have subscribed to the **general **topic above, let’s get the URL for that topic.
```

Comment:
```suggestion
7. You can get the Callback URL by visiting the Topics tab and selecting the topic you require. Since we have subscribed to the **general** topic above, let’s get the URL for that topic.
```